source/includes/language-compatibility-table-java.rst

```diff
@@ -4,22 +4,34 @@
    :class: compatibility-large
 
    * - Java Driver Version
+     - Java 21
      - Java 17
-     - Java 11 [#lts-note]_
+     - Java 11
      - Java 8
      - Java 7
      - Java 6
      - Java 5
 
+   * - 5.0
```

Comment:
We haven't released 5.0 yet.  Do we want to list it here already?

Reply:
This PR is against `master`, which users see as "upcoming" in the dropdown menu. The `current` version (4.11) will still only have instructions on downloading 4.11. In general, our policy is that it is fine for the compatibility table to show future versions compared to the most recent available version.